google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerRpcMetrics.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.spanner;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.LongCounter;
+import io.opentelemetry.api.metrics.LongHistogram;
+import io.opentelemetry.api.metrics.Meter;
+
+public class SpannerRpcMetrics {
```

Comment:
Does this class need to be public? (See also questions on the methods below).

If it is public for technical reasons, for example because we need to access it from a different package within the client library, but users should normally not access it, then we should add an `@InternalApi` annotation to the class.

Reply:
This needs to be public, as it needs to be used in HeaderInterceptor class which is in different package. Added the @InetrnalApi annotation to it.